microsoft/testsuites/core/boot.py

```diff
@@ -0,0 +1,86 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from pathlib import Path
+
+from assertpy.assertpy import assert_that
+
+from lisa import (
+    Logger,
+    RemoteNode,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    simple_requirement,
+)
+from lisa.features import SerialConsole
+from lisa.operating_system import Redhat
+from lisa.tools import Uname
+from lisa.util import LisaException, SkippedException
+from lisa.util.shell import wait_tcp_port_ready
+
+
+@TestSuiteMetadata(
+    area="core",
+    category="functional",
+    description="""
+    This test suite is to test VM working well after updating on VM and rebooting.
+    """,
+)
+class Boot(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This test case will
```

Comment:
Add explanation in test spec, why this test case work on Redhat only.

Reply:
this case was added by redhat, I also did some check, there is no debug kernel package for ubuntu, just debug symbol.

Reply:
So, add description in code. Others may have the same question.

Reply:
added